src/rust/ensogl/src/display/shape/primitive/glsl/color.glsl

```diff
@@ -8,6 +8,9 @@
 ///   - A Rgba struct which contains `vec4 raw` field for each component.
 ///   - A rich set of smart constructors for each of the types.
 ///   - A rich set of simple conversions between them.
+///
+/// All component values are in range 0.0..1.0.
```

Comment:
@wdanilo Can you confirm that this is correct for all colors?

Reply:
It could not be correct for the hue! If so (needs a check), could you update the docs here please? Thanks for catching1

Reply:
No there is no bug (that I encountered), this was just for reference, so someone reading this code knows the covnentions right away.